exercises/concept/lasagna/lasagna.js

```diff
@@ -0,0 +1,69 @@
+// @ts-check
```

Comment:
While I really like everything that's being said in these comments, my only concern would be that people would have to scroll down a long way before seeing any actual code when working on this in the browser. But that might not be an issue 🤷

Reply:
Eh. I'm not worried about that. I rather have it in their face the first 3 exercises, than adding it without context and it "just working".

Reply:
I can see how that particularly makes sense for JS given the large number of people relatively new to programming.

Reply:
Yeah. Trying to find the fine line between those two things.